bin/grant-server/main.go

```diff
@@ -114,6 +114,19 @@ func setupRouter(ctx context.Context, logger *logrus.Logger) (context.Context, *
 		r.Mount("/v2/attestations/safetynet", proxyRouter)
 	}
 
+	go func() {
+		ticker := time.NewTicker(5 * time.Second)
+		for {
+			attempted, err := promotionService.RunNextClaimJob(ctx)
+			if err != nil {
+				raven.CaptureErrorAndWait(err, nil)
+			}
+			if !attempted {
```

Comment:
i'm a little worried about us spinning here if we get an error and `attempted = true`. maybe we should delay (by waiting for ticker) this in case of either `!attempted || err != nil` ?